identity/app/views/profile/deletion/accountDeletionForm.scala.html

```diff
@@ -0,0 +1,142 @@
+@import idapiclient.responses.Error
```

Comment:
To make this PR diff better you could do a `git mv <source> <destination>` and git will show that you've just renamed the file